sentry_sdk/integrations/redis/_sync_common.py

```diff
@@ -42,13 +42,21 @@ def sentry_patched_execute(self, *args, **kwargs):
             origin=SPAN_ORIGIN,
         ) as span:
             with capture_internal_exceptions():
+                command_seq = None
+                if getattr(self, "_execution_strategy", None) and getattr(
+                    self._execution_strategy, "command_queue", None
+                ):
+                    command_seq = self._execution_strategy.command_queue
+                else:
+                    command_seq = self.command_stack
```

Comment:
[nit] `try`/`except` might be more succinct and readable here – although I am also totally fine with keeping this as is.

```suggestion
                try:
                    command_seq = self._execution_strategy.command_queue
                except AttributeError:
                    command_seq = self.command_stack
```